Setup.hs

```diff
@@ -0,0 +1,37 @@
+import Distribution.Simple
+import Distribution.Simple.Setup (BuildFlags)
+
+import Distribution.Simple.LocalBuildInfo
+    ( LocalBuildInfo
+    , buildDir
+    )
+
+import Distribution.PackageDescription
+    ( PackageDescription
+    , exeName
+    , executables
+    )
+
+import Control.Monad (when)
+import Data.Maybe (isJust, fromJust)
+import Data.List (find, isPrefixOf)
+import System.Directory (copyFile)
+import System.FilePath ((</>))
+
+main :: IO ()
+main = do
+  defaultMainWithHooks $ simpleUserHooks { postBuild = copyBinary }
+
+copyBinary :: Args -> BuildFlags -> PackageDescription -> LocalBuildInfo -> IO ()
+copyBinary _ _ desc buildInfo = do
+  let exe = findExecutable "site" desc
+
+  when (isJust exe) $ do
+    let binaryName = fromJust exe
```

Comment:
I knew about `fromJust` being unsafe, but I kinda like the explicitness here about it being a noop if we don't have the executable.

Honestly, I've been thinking about this function since I pushed this, and it _could_ just be

```haskell
do
    let exe = "site"
    buildDir buildInfo
    putStrLn $ "Copying executable '" ++ exe ++ "' to current directory..."
    copyFile (dir </> exe </> binaryName) binaryName
```

The whole `findExecutable` bit is probably overly safe.